lms/djangoapps/course_home_api/urls.py

```diff
@@ -38,3 +39,12 @@
         name='course-home-outline-tab'
     ),
 ]
+
+# Progress Tab URLs
+urlpatterns += [
+    re_path(
+        r'v1/progress/{}'.format(settings.COURSE_KEY_PATTERN),
+        ProgressTabView.as_view(),
+        name='course-home-progress-tab'
```

Comment:
i think the convention for these url names is to use underscores instead of dashes.